src/Query.ts

```diff
@@ -294,222 +297,200 @@ export class Query {
       },
     });
 
-    const filter =
-      searchProperty !== null && searchProperty.length !== 0
-        ? { searchProperty: { $elemMatch: { $in: searchProperty } } }
-        : { includeInGlobalSearch: true };
-
-    const compound: { should: any[]; must?: any[]; minimumShouldMatch: number } = {
+    const compound: { should: any[]; must?: any[]; filter?: any[]; minimumShouldMatch: number } = {
       should: parts,
       minimumShouldMatch: 1,
     };
 
+    // if (Object.keys(this.filters).length) {
+    //   compound['filter'] = compound['filter'] || [];
+    //   constructAggFilters(compound['filter'], this.filters);
+    // }
```

Comment:
(Just curious) If we don't plan on supporting this, should we remove the code?

Reply:
good point. i dont want filters. i'll remove the remnant filter logic and add you to PR 👍